tools/build-manylinux.sh

```diff
@@ -44,16 +50,16 @@ fi
 #
 
 echo "# Installing basic system dependencies"
-yum install -y zlib-devel gcc-c++
+yum install -y zlib-devel gcc-c++ python3 curl-devel perl-IPC-Cmd perl-Pod-Html
 
 echo "# Building librdkafka ${LIBRDKAFKA_VERSION}"
 $(dirname $0)/bootstrap-librdkafka.sh --require-ssl ${LIBRDKAFKA_VERSION} /usr
 
 # Compile wheels
 echo "# Compile"
-for PYBIN in /opt/python/*/bin; do
+for PYBIN in /opt/python/cp*/bin; do
     echo "## Compiling $PYBIN"
-    CFLAGS="-Werror -Wno-strict-aliasing -Wno-parentheses" \
+    CFLAGS="-Wno-strict-aliasing -Wno-parentheses" \
```

Comment:
Why this, there were other warnings that caused an error?

Reply:
Yes, with Python 3.9+ there were warnings of some header file being deprecated.

Reply:
please keep -Werror, let's see what they are and find a way to silence those warnings instead. Probably we want to upgrade those functions when the minimum Py version we support contains the alternative way to do that thing.

Reply:
Okay, let me try with `-Werror`.

Reply:
I've updated this. From Python 3.7+ `PyEval_InitThreads()` is a NO-OP and Python automatically calls this, so added a check to only call this if version is less than 3.9.

Taken reference from https://sourceware.org/pipermail/gdb-patches/2020-April/167630.html